pkg/kv/import.go

```diff
@@ -0,0 +1,70 @@
+package kv
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/treeverse/lakefs/pkg/logging"
+)
+
+var ErrInvalidFormat = errors.New("invalid format")
+
+// Header contains metadata information for import / export file
+type Header struct {
+	LakeFSVersion   string
+	PackageName     string
+	DBSchemaVersion int
+	CreatedAt       time.Time
+}
+
+func Import(ctx context.Context, reader io.Reader, store Store) error {
+	jd := json.NewDecoder(reader)
+	// Read header
+	var header Header
+	if err := jd.Decode(&header); err != nil {
+		if errors.Is(err, io.EOF) {
+			return fmt.Errorf("empty file: %w", ErrInvalidFormat)
+		} else {
+			return fmt.Errorf("decoding header: %w", err)
+		}
+	}
+	// Decode does not return error onm missing data / incompatible format
+	if header == (Header{}) {
+		return fmt.Errorf("bad header format: %w", ErrInvalidFormat)
+	}
+
+	// TODO(niro): Add validation the header in the future
+	logging.Default().WithFields(logging.Fields{
+		"package name":   header.PackageName,
+		"lakeFS version": header.LakeFSVersion,
+		"DB Version":     header.DBSchemaVersion,
+		"CreatedAt":      header.CreatedAt,
```

Comment:
prefer to process snake case later when parse structure log

```suggestion
		"package_name":   header.PackageName,
		"lakefs_version": header.LakeFSVersion,
		"db_schema_version":     header.DBSchemaVersion,
		"created_at":      header.CreatedAt,
```